pkg/experimentcalculator/experimentcalc/experiment_calculator.go

```diff
@@ -129,6 +129,11 @@ func (e ExperimentCalculator) Run(ctx context.Context, request *calculator.Batch
 				// experiment not started yet
 				continue
 			}
+			if ex.Status == experiment.Experiment_STOPPED &&
+				now.Unix()-ex.StopAt > 2*day {
+				// experiment stopped and 2 days passed
```

Comment:
Please add the following comment.

```
Because the evaluation and goal events may be sent with a delay for many reasons from the client side, we still calculate the results for two days after it stopped.
```

Reply:
Done!